### PR TITLE
kata-deploy: install oras using release artefacts on ppc64le

### PR DESCRIPTION
**Pull request by Amulyam24**
We are currently building Oras from source on ppc64le. Now that they offically release the artefacts for power, consume them to install Oras.

This PR also updates oras version to `1.2.0`

Fixes: #9213

### CODE REVIEW

**Comment by stevenhorsman** [comment]
@Amulyam24 - Does you have an understanding of when oras 1.20 might get a full release?

**Comment by Amulyam24** [comment]
> @Amulyam24 - Do you have an understanding of when oras 1.20 might get a full release?

Hi @stevenhorsman, I'm not sure. The last time I checked, it was mid of Jan. I pinged them again.

**Review by cmaf — APPROVED** [review]
Thanks @Amulyam24

**Comment by Amulyam24** [comment]
> > @Amulyam24 - Do you have an understanding of when oras 1.20 might get a full release?
> 
> Hi @stevenhorsman, I'm not sure. The last time I checked, it was mid of Jan. I pinged them again.

@stevenhorsman, PTAL -- https://github.com/oras-project/oras/discussions/1119#discussioncomment-8703886

**Comment by stevenhorsman** [comment]
> @stevenhorsman, PTAL -- https://github.com/oras-project/oras/discussions/1119#discussioncomment-8703886

Great - thanks for the info. Do you think it's okay for us to wait a month to pick up the proper release before we merge this rather than switching to the beta?

**Comment by Amulyam24** [comment]
> > @stevenhorsman, PTAL -- [oras-project/oras#1119 (reply in thread)](https://github.com/oras-project/oras/discussions/1119#discussioncomment-8703886)
> 
> Great - thanks for the info. Do you think it's okay for us to wait a month to pick up the proper release before we merge this rather than switching to the beta?

yes, that should be fine. Okay to put in on hold 👍🏼

**Comment by stevenhorsman** [comment]
Hi @Amulyam24 - given that oras 1.2 has just released (better late than never): https://github.com/oras-project/oras/releases/tag/v1.2.0 - are you okay to update and rebase this PR now? Thanks!

**Comment by Amulyam24** [comment]
> Hi @Amulyam24 - given that oras 1.2 has just released (better late than never): https://github.com/oras-project/oras/releases/tag/v1.2.0 - are you okay to update and rebase this PR now? Thanks!

Hi @stevenhorsman, Thanks for the update, rebased the PR!

**Review by stevenhorsman — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**
